src/cursor/cur_std.c

```diff
@@ -651,6 +651,12 @@ __wt_cursor_cache(WT_CURSOR *cursor, WT_DATA_HANDLE *dhandle)
 
     WT_TRET(cursor->reset(cursor));
 
+    /* 
+     * The cursor reset only clears bounds when called externally. Therefore explicitly clear the 
```

Comment:
This should be worded differently.

"Cursor reset clears bounds on cursors when called externally, we need to clear the bounds manually when we cache a cursor."